packages/graphback-datasync/src/DataSyncPlugin.ts

```diff
@@ -59,6 +61,13 @@ export class DataSyncPlugin extends GraphbackPlugin {
       console.warn("No DataSync Models detected, ensure that your models are properly annotated.")
     }
 
+    if (modelConflictConfigSet.size !== 0) {
+      console.log("The following models from conflictConfig could not be found, consider adding them to the schema and/or adding the @datasync annotation if you have not done so: ");
+      modelConflictConfigSet.forEach((modelName: string) => {
+        console.log(modelName)
```

Comment:
Remove

Reply:
This is the check for invalid conflict key #1930, should I remove it?

Reply:
Sorry for not being clear. 

Having single console log is fine.

${modelConflictConfigSet.join("\n")}